planner-api/src/main/java/pl/edu/agh/kuce/planner/auth/Current.java

```diff
@@ -0,0 +1,13 @@
+package pl.edu.agh.kuce.planner.auth;
+
+import org.springframework.security.core.annotation.CurrentSecurityContext;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.PARAMETER })
```

Comment:
Would be possible to add a comment about these lines? Tbh I am not sure what are they doing and I think a comment here can be valuable in the future.